source/Playnite.Common.NetStandard/EasyAntiCheat.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Newtonsoft.Json;
+using Playnite.Common;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Playnite
+{
+    public class EasyAntiCheatLauncherSettings
+    {
+        [JsonProperty(PropertyName = "title")]
+        public string Title { get; set; }
+
+        [JsonProperty(PropertyName = "executable")]
+        public string Executable { get; set; }
+
+        [JsonProperty(PropertyName = "logo_position")]
+        public string LogoPosition { get; set; }
+
+        [JsonProperty(PropertyName = "parameters")]
+        public string Parameters { get; set; }
+
+        [JsonProperty(PropertyName = "use_cmdline_parameters")]
+        public string UseCmdlineParameters { get; set; }
+
+        [JsonProperty(PropertyName = "working_directory")]
+        public string WorkingDirectory { get; set; }
+
+        [JsonProperty(PropertyName = "wait_for_game_process_exit")]
+        public string WaitForGameProcessExit { get; set; }
+
+        [JsonProperty(PropertyName = "hide_splash_screen")]
+        public string HideSplashScreen { get; set; }
+
+        [JsonProperty(PropertyName = "ide_ui_controls")]
+        public string IdeUiControls { get; set; }
+    }
+
```

Comment:
split in two files

Reply:
That was like that in the Playnite code. There are a thousand files with several classes inside, so I think we'll be better leaving this untouched.